src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -28,6 +28,7 @@ public partial class CSharpCompilation
         /// </summary>
         private Symbol[] _lazyWellKnownTypeMembers;
 
+        private bool _usesNullableAttributes;
         private int _needsGeneratedAttributes;
         private bool _needsGeneratedAttributes_IsFrozen;
```

Comment:
nit: should this field be renamed, since now used for freezing both "needsGenerated" and "usesNullable"?

Reply:
Yes, I considered renaming the field but didn't find a better name.

---
In reply to: [301354830](https://github.com/dotnet/roslyn/pull/37019#discussion_r301354830) [](ancestors = 301354830)